Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -347,6 +347,7 @@ class BrowserViewController: UIViewController {
         urlBar = URLBarView()
         */
         urlBar = CliqzURLBar()
+		NotificationCenter.default.addObserver(self, selector: #selector(urlBarDidPressPageCliqzOptions), name: URLBarDidPressPageOptionsNotification, object: nil)
```

Comment:
could you fix the indentation of this line

Reply:
The indentation is fine, github  messes up...